addons/mass_mailing_crm/data/mass_mailing_demo.xml

```diff
@@ -4,7 +4,7 @@
     <record id="mass_mail_lead_0" model="mailing.mailing">
         <field name="name">Lead Recall</field>
         <field name="subject">We want to hear from you !</field>
-        <field name="state">in_queue</field>
+        <field name="state">done</field>
```

Comment:
Does that imply other things?
Such as not processing the mailing and thus not displaying any statistics for it?

Reply:
Bloups?

Reply:
as discussed, I dropped the commit